### DISCUSSION TITLE
İhmal Takibi İçin Yıkılan Binalar Veritabanı

### DISCUSSION THREAD

**Discussion by selamikarakas** [acikkaynak/deprem-yardim-projesi#28] [Ideas]
Yıkılan binaların yapımında ihmal var mı diye araştırılmasının kolaylaşması için böyle bir database ihtiyacı olduğunu düşünüyorum. İkinci aşamada bu binaların yapım şirketlerinin araştırılmasını ve bu database'e beslenmesinin çok yararlı olacağını düşünüyorum. Data'nın nasıl toplanacağı ile ilgili fikirlerinizi bekliyorum.

**Comment by canerozer** [comment]
Merhaba Selami,

Bu konuda belki uzaktan algılama sistemlerinin faydalı olacağını düşünüyorum. Şöyleki, [Planet](https://www.planet.com/) veya [Sentinel-Hub](sentinel-hub.com) gibi şirketler uzun süreli görüntü kayıtlarına erişim sağlatabiliyor. Deprem öncesi ve sonrası duruma göre hangi binaların yıkılıp yıkılmadığını tespit edebiliriz bu sitelerin kayıtlarını kullanarak. Yıkıldığını tespit ettiğimiz binaların da açık adres bilgisini kayıt ettiğimiz zaman elimizde yıkılmış binaların adreslerinin bulunduğu bir veritabanı bulunmuş olur ki ilk aşamada işimizi görebilir.

Burada sadece şöyle bir zorluk var benim düşündüğüm, bu uzaktan algılama vasıtasıyla elde edilen görüntülerin çok yüksek çözünürlüklü olması gerekiyor, başka türlü anlaşılması ve sonuçların irdelenmesi zor olabiliyor. Mesela ben bugün Sentinel-Hub üzerinden Kahramanmaraş'ın bir bölümünün deprem öncesi ve sonrası durumlarını karşılaştırma maksadıyla iki görüntü çektim sistemlerinden ve şu şekilde bir sonuçlar çıktı. Planet kullanarak daha yüksek çözünürlüklü fotoğraflar çıkarabilmek mümkün diyorlardı, bir denemeye çalışacağım onu da.
![2023-02-09-00_00_2023-02-09-23_59_Sentinel-2_L2A_True_color](https://user-images.githubusercontent.com/9446152/218194993-159bdf6f-1879-4e7f-a5ab-8fa16be6f2f6.jpg)
![2023-01-25-00_00_2023-01-25-23_59_Sentinel-2_L2A_True_color](https://user-images.githubusercontent.com/9446152/218195019-92cd964e-7fab-4b0b-88fc-ac3cdd353ecd.jpg)

**Reply by sercangoger** [reply]
güzel proje burada kalsın. umarım görürler buradaki arkadaşlar.

**Comment by erogluegemen** [comment]
bu data toplanır ise benim de basit bir websitesi fikrim var. 
İleride insanlar ev alırken, alacakları evi yapan şirketi bu websitesi üzerinden sorgulayabilecek.
Eğer şirket db'de ihmalden dolayı yer alıyosa kullanıya uyarı verecek olan basit bir uygulama.
Aynı şekilde datanın toplanması yönünde fikri olan varsa duymak isterim. 🙏🏻

**Comment by FundaOzmen** [comment]
Fikrine ek olarak bu evin jeofiziki durumu, müteahhit bilgileri, jeofiziki durumu, bulunduğu bölge daha önce neydi? Tarım alanı mi, orman mi? Vb .. bilgileri görebilecek mesela ...

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by furkan-cat** [reply]
Keyifli bir fikir. Gerçekten ev almadan evi satanın yağlı ballı laflarıdan ziyade iinsanların somut veriler üzerinden değerlendirme yapması gerek. Hatta bunun e-devlet içerisinde bir sistem olması gerektiğini düşünüyorum.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
